<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(235, 235, 235);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="cod_cnd_pgto">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>89</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="des_cnd_pgto">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qtd_dias">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblcod">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CÓDIGO COND.PGT</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblqtddias">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>QUANTIDADE DE DIAS</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbldesc">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DESCRIÇÃO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="incluir">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>75</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>75</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>INCLUIR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modificar">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>MODIFICAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="excluir">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>EXCLUIR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="consultar">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CONSULTAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="relatorio">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>RELATORIO</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbltitulo">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(158, 255, 151);</string>
    </property>
    <property name="text">
     <string>CADASTRO DE CONDIÇÃO DE PAGAMENTO</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>745</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
